lib/ansible/modules/git.py

```diff
@@ -265,26 +265,26 @@
 
 - name: Checkout a github repo and use refspec to fetch all pull requests
   ansible.builtin.git:
-    repo: https://github.com/ansible/ansible-examples.git
-    dest: /src/ansible-examples
+    repo: git@github.com:mylogin/hello.git
```

Comment:
This example (and the others) should use a repository within the `ansible` org.

```suggestion
    repo: git@github.com:ansible/ansible.git
```

Reply:
I actually used [this](https://github.com/ansible/ansible/blob/a8e4106c4748995ff4e6b91e84d7692b654b721f/lib/ansible/modules/git.py#L250) as a reference in order to replace the other examples to keep consistency. Replacing it too would be out of scope for this PR? What about examples like [this](https://github.com/ansible/ansible/blob/a8e4106c4748995ff4e6b91e84d7692b654b721f/lib/ansible/modules/git.py#L244) one?

Reply:
Updating the other non-ansible references in the examples would be good to maintain consistency.

Reply:
Great! Done.